### ISSUE TITLE
[BUG] Molecular Function Enrichment doesn't work

### ISSUE THREAD

**Issue by gdefazio** [MaayanLab/enrichr_issues#48]
**Describe the bug**
Molecular Function (MF) Enrichment (by using GO_Molecular_Function_2021, GO_Molecular_Function_2020 or GO_Molecular_Function_2019) for both enrichr R package and API doesn't return erichment analysis MF results 
for 454 genes sets. 
By contrast, the MF enrichment correctly works when I use the enrichr web site with some of the same genes sets. 

On the other hand, when I use the test genes set provided by enrichr API guide, it correctly works.

**Comment by AviMaayan** [comment]
Hi @gdefazio there might be an issue with the formatting of your gene set. Can you try dividing it into two or more parts and see if it works for half of the genes in your set?

**Comment by gdefazio** [comment]
I don't think that the problem is the formatting because when I used the same sets with the same formatting for BP and CC with the R package and API, I got the results. By contrast, the same sets with the same formatting returned empty tables for MF.

**Comment by AviMaayan** [comment]
Is it possible that there is no overlap, or no significant results for MF?

**Comment by gdefazio** [comment]
It is possible but improbable because I tested 454 gene sets. Moreover, I tested some of them on the website that showed significant results for MF.
